docs/ops/pooling/MaxPool_1.md

```diff
@@ -63,24 +63,192 @@
   * **Type**: string
   * **Default value**: *explicit*
   * **Required**: *no*
-  * **Note**: *pads_begin* and *pads_end* attributes are ignored when *auto_pad* is specified.
+  * **Note**: *pads_begin* and *pads_end* attributes are ignored when *auto_pad* is not equal to explicit.
 
 **Inputs**:
 
-*   **1**: 3D, 4D or 5D input tensor. Required.
+*   **1**: 3D, 4D or 5D input tensor of type T. Required.
 
-**Mathematical Formulation**
+**Outputs**:
+  * **1**: Input shape can be either `[N, C, H]`, `[N, C, H, W]` or `[N, C, H, W, D]`. Then the corresponding output shape will be `[N, C, H_out]`, `[N, C, H_out, W_out]` or `[N, C, H_out, W_out, D_out]`. Output tensor has the same data type as input tensor.
+
+**Types**
 
-    \f[
-    output_{j} = max(x_{0}, ..., x_{i})
-    \f]
+* *T*: `uint64`, `int32`, `int64`, `float16`, `float32`.
```

Comment:
`uint64` and `int64` are not supported for sure. Also, we probably need to add `bf16`. @alexey-varyzgin  could you comment on that?

Reply:
@lazarevevgeny should we also use the tick character with types, e.g. 'int32_t' ? I didn't use it in my changes, just want to establish common practice.

Reply:
Use tick with types. Recent operations like docs/ops/sequence/CTCGreedyDecoderSeqLen_6.md, docs/ops/movement/GatherElements_6.md and others use them.

Reply:
BTW, we don't specify `int32_t` and similar. We use just `int32`, `int64` etc.

Reply:
I run SLT for all data types and int32, int64, bf16, f16, f32, uint64 passed

Reply:
I set supported types as "floating point or integer type" as @jdanieck told me to do.